lib/driver.js

```diff
@@ -274,7 +275,8 @@ class XCUITestDriver extends BaseDriver {
       log.info(`Xcode version set to '${this.xcodeVersion.versionString}' ${tools}`);
 
       this.iosSdkVersion = await getAndCheckIosSdkVersion();
-      log.info(`iOS SDK Version set to '${this.iosSdkVersion}'`);
+      this.opts.iosSdkVersion = this.iosSdkVersion; // Pass to xcodebuild
```

Comment:
`this.iosSdkVersion` is shared with ios-driver. Thus, I've added it as a part of `opts` since the `opts` is an argument of WDA.